docassemble/StudentEvaluations/data/questions/massachusetts_educational_evaluations_basic_information.yml

```diff
@@ -393,6 +393,13 @@ fields:
 validation code: |
   school.name.text = school.address.name
 
+
```

Comment:
This is commented out code

Reply:
Do you mean I should comment that code out? Those I was able to take out the gender question and fix the issue with the address of the school being repeated 2 times in the document

Reply:
Lines 397 - 402 here are commented out and therefore not active code. It shows as being added to the yml file not removed. Is there a reason for adding this commented out code?

Reply:
No. I will remove it now.